docs/data/xrootd-authorization.md

```diff
@@ -0,0 +1,141 @@
+Configuring XRootD Authorization
+================================
+
+There are several authorization options in XRootD available through the security plugins. 
+In this document, we will cover the security option supported in the OSG:
+
+-   [`xrootd-lcmaps`](#recommended-security-option-xrootd-lcmaps-authorization): Using the LCMAPS callout to use
+    X509-based authentication
```

Comment:
Seems a little silly to have a list of a single item

> In this document, we will cover the `xrootd-lcmaps` security option supported in the OSG.

Reply:
Do you want to drop the "Using the LCMAPS callout to use
    X509-based authentication" text, then?

Reply:
Assuming yes, and updating.

Reply:
How about we tack on [this sentence](https://github.com/opensciencegrid/docs/pull/458/files#diff-ad770d3b827d69a2612a4bc062483657R85) here and reword the intro paragraph of the "Enabling xrootd-lcmaps authorization"